docs/reference/setup/install/windows.asciidoc

```diff
@@ -68,6 +68,8 @@ the command line, using the `-E` syntax as follows:
 
 NOTE: Values that contain spaces must be surrounded with quotes.  For instance `-Epath.logs="C:\My Logs\logs"`.
 
+NOTE: Values in the `elasticsearch.yml` file that contain backslashes must be escaped with `\\`.  For instance `path.logs="C:\\My Logs\\logs"`.
+
```

Comment:
How about combining this NOTE with the previous one.  Too many NOTEs on a page and people don't look at any of them